tests/projects/test_projects_cli.py

```diff
@@ -17,7 +17,7 @@ def test_run_local():
         with update_temp_env({mlflow.tracking._TRACKING_URI_ENV_VAR: tmp.path()}):
             excitement_arg = 2
             res = invoke_cli_runner(cli.run, [TEST_PROJECT_DIR, "-e", "greeter", "-P",
-                                              "greeting=hi", "-P", "name=friend",
+                                              "greeting=hi", "--no-conda", "-P", "name=friend",
```

Comment:
do we want to test this with and without conda?

Reply:
good question, we do ideally want to test with conda - however the conda installation currently runs so slowly that the test times out from not receiving any new output (or if we change our test config to continuously write conda's output, we exceed Travis's 4 MB limit on test output). if you're fine with it, would be nice to merge this for now & I'll continue to debug the test with conda - the conda installation shouldn't take that long in the first place IMO.

Reply:
+1